docs/architecture/adr-013-non-interactive-default-rules-for-zero-padding.md

```diff
@@ -68,9 +80,9 @@ Here is a diagram of the worst-case padding for a threshold of 16 for the square
 
 ### Additional Remarks
 
-If the threshold is bigger than `MinSquareSize` for a particular blob then the blob will be aligned to the index of the `MinSquareSize` of the blob. This would prevent some blob size ranges to have higher padding than they had before this change. So the real new non-interactive default rules would be:
+If the threshold is bigger than `MinSquareSize(blob)` then the blob will be aligned to the index of the `MinSquareSize(blob)`. This would prevent some blob size ranges to have higher padding than they had before this change. So the real new non-interactive default rules would be:
 
-Blobs start at an index that is equal to a multiple of the blob length divided by `MaxSquareSize` rounded up. If this index is larger than the `MinSquareSize` of the blob then the blob starts at the index of the `MinSquareSize`.
+Blobs start at an index that is equal to a multiple of the blob length divided by `MaxSquareSize` rounded up. If this index is larger than the `MinSquareSize(blob)` then the blob starts at the index that is a multiple of of the `MinSquareSize(blob)`.
```

Comment:
Maybe we reference that where we explain the new non-interactive default rules? See my comment above

Reply:
It is good. You are introducing subtree root width as a variable name, but explaining what it means in words may also make sense as it is the first mention in the ADR.

Reply:
Was [this footnote](https://github.com/celestiaorg/celestia-app/pull/1959/files#diff-d55bfa04f022c5fcffa40f2fe89ff8f7047a6367f2f0ba73eab3e40c3923d5eaR105) not easy to discover? or are there other words you would use to define it?

Reply:
I'm going to merge b/c two approvals but happy to amend the footnote or describe it differently in a follow-up PR. Thanks for your reviews!